src/components/Nav/Nav.js

```diff
@@ -1,7 +1,106 @@
-import React from 'react';
+import { React, useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import './Nav.scss';
 
 const Nav = () => {
-  return <div>Nav</div>;
+  const [navCategory, setNavCategory] = useState([]);
+  const [isHovering, setIsHovering] = useState(0);
```

Comment:
hover된 상태인지 아닌지를 나타내는 상태값이라면 boolean 값으로 관리해주세요! 
```suggestion
  const [isHovering, setIsHovering] = useState(false);
```